sfm/message_consumer/sfm_ui_consumer.py

```diff
@@ -98,6 +98,55 @@ def _on_harvest_status_message(self):
                     except ObjectDoesNotExist:
                         HarvestStat.objects.create(item=item, harvest=harvest, count=count, harvest_date=day)
 
+        # Send email if completed and failed or has messages
+        if harvest.status == Harvest.FAILURE or (
+                harvest.status == Harvest.SUCCESS and (harvest.infos or harvest.warnings or harvest.errors)):
+
+            # Get emails for group members
+            receiver_emails = []
+            for user in harvest.collection.collection_set.group.user_set.all():
+                if user.email:
+                    receiver_emails.append(user.email)
+
+            if receiver_emails:
+                harvest_url = 'http://{}{}'.format(Site.objects.get_current().domain,
+                                                   reverse('harvest_detail', args=(harvest.id,)))
+
+                # Send Status mail
+                if settings.PERFORM_EMAILS:
+                    if harvest.status == Harvest.SUCCESS:
+                        mail_subject = u"SFM Harvest for {} completed successfully, but has messages".format(
+                            harvest.collection.name)
+                        mail_message = u"The harvest for {} ({}) completed successfully.".format(
```

Comment:
@justinlittman is there a reason the message doesn't also say "but has messages"? When viewing the email, it would be useful to have the full info in the body.